trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -538,6 +538,7 @@ def _add_lines_to_spectrum(self, field_data, use_peculiar_velocity,
             # light ray that is deposited to the final spectrum 
             if store_observables:
                 tau_ray = np.zeros(cdens.size)
+                current_tau_field = 0*self.tau_field
```

Comment:
Maybe include a comment here telling us what `current_tau_field` represents?  Just a single line that it is meant to track the tau from an individual transition or something?